src/core/observables/CylindricalLBProfileObservable.hpp

```diff
@@ -47,16 +47,15 @@ class CylindricalLBProfileObservable : public CylindricalProfileObservable {
         limits[0], limits[1], limits[2], n_bins[0], n_bins[1], n_bins[2],
         sampling_density);
     for (auto &p : sampling_positions) {
-      double theta;
-      Utils::Vector3d rotation_axis;
       auto p_cart = Utils::transform_coordinate_cylinder_to_cartesian(
           p, Utils::Vector3d{{0.0, 0.0, 1.0}});
       // We have to rotate the coordinates since the utils function assumes
       // z-axis symmetry.
-      std::tie(theta, rotation_axis) =
-          Utils::rotation_params(Utils::Vector3d{{0.0, 0.0, 1.0}}, axis);
+      constexpr Utils::Vector3d z_axis{{0.0, 0.0, 1.0}};
```

Comment:
why do we still manually rotate at the call site? I thought the idea was to include that into the transformation? Then you can also test it properly.